CHANGELOG.md

```
@@ -1,3 +1,130 @@
+## 1.3.0 - 2025-09-02
+
+#### 🔥 Major Feature: Comprehensive Bubble Chart Support
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)